### DISCUSSION TITLE
Can't log in

### DISCUSSION THREAD

**Discussion by indy2001** [flybywiresim/aircraft#4113] [Help]
Until this past weekend everything was fine, but since then I haven't been able to update anything and I don't have any A32NX aircraft any longer. I have tried installing all 3 types of files (Stable, Development, and Custom FBW). They seem to install OK, but during the Decompressing after the model module install, the installer hangs up and must be shut down through Task Manager. I have tried from within the installer, and also downloading each file individually, but the same thing happens every time. I have deleted the A32NX folder from the Community folder numerous times, but that doesn't help. In fact, the installer tells me I have the Stable version installed when there is no folder there! Also, all my A320 liveries now have AIRCRAFT.UI_MANUFACTURER above the airline. Finally, the basic A320 doesn't work because all of the gauges are blank when it starts up.

Any help you can give would be very much appreciated.

**Comment by mojoair** [comment]
1) Uninstall the FlyByWire installer. Also removing the fbw_installer folder in C:\Users\(Yourusername)\AppData\Local\ incase there are any remaining files.

2) Completely empty your community folder (back it up onto a different folder).

3) Open MSFS and go to  Profile -> Content Manager. Delete the A320Neo and all associated default liveries you may have installed. Quit and Restart MSFS. Redownload and install the Default Airbus through the Content Manager again and see if the default A320 works.

3) Manually download the A32NX zip file from the main Github page. You can find them under "Traditional Download Methods". I recommend the Stable version to start with.

4) Extract the downloaded zip file. Once extracted, you should end up with a folder "A32NX". This is the folder you move to the Community folder. You should not have any issues extracting this zip file with say 7-Zip or the default windows extraction tool. If you do, this is an issue with your computer, not FBW or MSFS.

5) If everything works, you can redownload the FBW Installer and give that a go from here on out.

It appears some default gauges/model files may have gotten corrupted and reinstalling the default aircraft should redownload these files.

**Reply by indy2001** [reply]
Thanks so much for your help. Steps 1, 2, and 3 worked perfectly. However, I had to "fiddle" with the A32NX installation a bit. I think something in my Community folder doesn't get along with the A32NX but I'm not sure which one. Anyway, I'm flying with the latest Custom FBW again, so I'm very happy. Thanks again.
